packages/http/src/__tests__/memory-leak-prevention.spec.ts

```diff
@@ -33,14 +33,12 @@ describe('Checks if memory leaks', () => {
 
     for (let i = 0; i < 5000; i++) {
       round(client);
-      if (i % 100 === 0 && global.gc) {
+      if (i % 100 === 0) {
```

Comment:
Help me out: I guess this test will fail if `global.gc()` doesn't exist, but why was it optional before?  (This might be a good place for a comment.)

Reply:
Apparently that was another change made to satisfy the @types/node@18 typings.

`global.gc` could not be undefined in @types/node@13 but it can in v18